### PR TITLE
Extract tau=1 height pressure

### PR DESCRIPTION
**Pull request by chonma0ctopus**
Added a module to extract the pressure at the height of tau = 1 (or another given value) at different wavelengths, and its unittest and example notebook.
Note that the module will return 0 if tau is too optically thin and does not reach tau=1.

### CODE REVIEW

File: src/exojax/spec/tau1height.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+"""Extract tau=1 height pressure at different wavelengths"""
+import numpy as np
+from scipy import interpolate
+
+def get_P_at_tauone(dtau, Parr, tauextracted=1.0):
```
Comment (HajimeKawahara) [inline src/exojax/spec/tau1height.py:5]: `get_P_at_tauone` -> `extract_pressure_at_tauone`? (from the comment)
or simply `pressure_at_given_opacity`? any good idea?

Comment (chonma0ctopus) [reply src/exojax/spec/tau1height.py:5]: Thanks for the suggestion! Okay, I'll choose `pressure_at_given_opacity`.

File: src/exojax/spec/tau1height.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"""Extract tau=1 height pressure at different wavelengths"""
```
Comment (HajimeKawahara) [inline src/exojax/spec/tau1height.py:1]: Not sure but, looks a bit rare to include "1" in the name of modules. I have no good idea but for instance
- tauinfo (assuming we include more analysis functions in it)
- opacitytools 
-  ... no good idea...

Comment (HajimeKawahara) [reply src/exojax/spec/tau1height.py:1]: i mean i talk about "tau1height.py" just in case

**Comment by HajimeKawahara** [comment]
Thanks! I had some comments for names.

File: src/exojax/spec/tau1height.py (hunk 2 above)
Comment (chonma0ctopus) [reply src/exojax/spec/tau1height.py:1]: Thanks! I'll adopt `opacitytools`.

**Comment by HajimeKawahara** [comment]
@chonma0ctopus 
What is `modit_issue277.py`?

**Comment by chonma0ctopus** [comment]
> @chonma0ctopus What is `modit_issue277.py`?

Sorry, I mis-pushed... never mind please.

**Comment by chonma0ctopus** [comment]
Now, it is fixed and all the suggested name changes have been done.

**Review by HajimeKawahara — APPROVED** [review]
Thanks! Cool figure!
image

**Pull request merged**